src/core/hle/service/hid/hid.cpp

```diff
@@ -192,10 +148,10 @@ void Module::UpdatePadCallback(std::uintptr_t user_data, s64 cycles_late) {
     TouchDataEntry& touch_entry = mem->touch.entries[mem->touch.index];
     bool pressed = false;
     float x, y;
-    std::tie(x, y, pressed) = touch_device->GetStatus();
+    /* std::tie(x, y, pressed) = touch_device->GetStatus();
```

Comment:
While kraken supports multitouch you can just use the first index here and ignore the rest or apply an average. The inputs are always ordered. This code is from emulated_console.cpp